app/src/main/java/com/kickstarter/viewmodels/BackingFragmentViewModel.kt

```diff
@@ -249,6 +259,18 @@ interface BackingFragmentViewModel {
             }
                 .map { it.second.userIsCreator(it.first.getValue()) }
 
+            backing
+                .map { it.paymentIncrements ?: emptyList() }
```

Comment:
i think we should check if payment increments is not null, or empty here, not just null.